Add spotbugs filter file

cmathiesen opened the pull request.

cmathiesen committed:
add spotbugs filter file

massdosage commented on src/main/resources/spotbugs/hotels-spotbugs-exclude.xml:
```diff
@@ -0,0 +1,10 @@
+<!-- these bugs/files/etc won't be reported by findbugs -->
```
Rename this file to `eg-spotbugs-exclude.xml`

massdosage replied:
findbugs -> spotbugs

massdosage reviewed (changes requested):
Bump minor version since we're adding a feature. Also please add a line to the changelog.

cmathiesen committed:
Add contributing.md, update changelog, correct naming errors

massdosage committed:
Full stop!

massdosage committed:
SNAPSHOT version

We never set release versions ourselves, this is done by the Maven release plugin at release time.

massdosage committed:
prep changelog for release after merge
